Docs: warn `parallelism > 1` doesn't work for migration procedures (#11417)
apache/iceberg: docs/docs/spark-procedures.md

| `backup_table_name` |  | string | Name of the table that will be retained as backup (defaults to `table_BACKUP_`) |
| `parallelism` |   | int | Number of threads to use for file reading (defaults to 1) |

!!! warning

>     There's a [known issue with `parallelism > 1`](https://github.com/apache/iceberg/issues/11147). We will fix it in next release.

Reviewer: ```suggestion
    There's a [known issue with `parallelism > 1`](https://github.com/apache/iceberg/issues/11147) that is scheduled to be fixed in the next release.
```

Warning : Files added by this method can be physically deleted by Iceberg operations

!!! warning

>     There's a [known issue with `parallelism > 1`](https://github.com/apache/iceberg/issues/11147). We will fix it in next release.

Reviewer: ```suggestion
    There's a [known issue with `parallelism > 1`](https://github.com/apache/iceberg/issues/11147) that is scheduled to be fixed in the next release.
```
